scripts/test/DynamicField/EditFieldRender.t

```diff
@@ -3370,9 +3371,14 @@ for my $Test (@Tests) {
     }
     if ( $Test->{Success} ) {
 
+        $FieldHTML->{Field} =~ s/^\n+//;
+        $FieldHTML->{Field} =~ s/\n+$//;
+        $FieldHTML->{Field} =~ s/\n{2,}/\n/g;
+
         # Heredocs always have the newline, even if it is not expected
         if ( $FieldHTML->{Field} !~ m{\n$} ) {
-            chomp $Test->{ExpectedResults}->{Field};
+            $Test->{ExpectedResults}->{Field} =~ s/^\n+//;
+            $Test->{ExpectedResults}->{Field} =~ s/\n+$//;
```

Comment:
Please change this. Comment out the old stuff, and make your adjustments all together with a preceding comment: "# TODO: Have a look at the newlines produced during template rendering. See Issue #..."